av1-rtp-spec.md

```diff
@@ -150,10 +150,13 @@ The structure is as follows.
 </code></pre>
 
 Z: MUST be set to 1 if the first OBU element is an OBU fragment that is a continuation of an OBU fragment from the previous packet, and MUST be set to 0 otherwise.
+{:& ParseTreatsContinuationFlagAsNotBeginningOfFrame,ParseTreatsNoContinuationFlagAsNotBeginningOfFrame}

```

Comment:
I guess this refer to the test
https://source.chromium.org/chromium/chromium/src/+/master:third_party/webrtc/modules/rtp_rtcp/source/video_rtp_depacketizer_av1_unittest.cc;l=41;drc=69679598e79799b864dad283bf1cab2f194995d9

how would reader or plugin know where to find more about the test?
could it help to use more descriptive name for the test, e.g.
webrtc.VideoRtpDepacketizerAv1Test.ParseTreatsContinuationFlagAsNotBeginningOfFrame

Reply:
In W3C specs, there is a link to the (WPT) test.  A descriptive name is better than an obscure one, but it should be possible to follow a link to the test.

Reply:
A test name is a chain of characters, so it can be a descriptive name. We just need to define a character or a sequence of characters that will be the delimiter between two or more tests.
Adding a link to the test is possible as well.
We may need to define some syntax to combine both a descriptive name together with a link to the test.

Reply:
> To make these annotations manageable by the Jekyll/markdown conversion process, it is necessary to add an extension

Do we know which Jekyll extension is needed? Only a limited set of extensions is supported by GitHub Pages, for security reasons. Also, is there a visual example of the desired annotation?